pages/api/prescriptions/create.ts

```diff
@@ -47,6 +48,40 @@ export default async function handler(
         }
       })
 
+      const patient = await prisma.patient.findUnique({
+        where: { id: patientId }
+      })
+
+      const user = await prisma.user.findUnique({
+        where: { id: patient?.userId }
+      })
+
+      const location = await prisma.location.findUnique({
+        where: { id: locationId }
+      })
```

Comment:
You should be able to consolidate this all into one query to the `user` table  using the `include` option for prisma queries https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#include.  You can also use the `findUniqueOrThrow` call to automatically handle an error if the patient doesn't exist.